starter-code/src/functions-and-arrays.js

```diff
@@ -60,6 +138,19 @@ var wordsFind = [
   'disobedience'
 ];
 
+function doesWordExist(wordsFind) {
```

Comment:
You would probably need to pass 2 parameters into this function in order to have a word to check if it exists in the array you pass in